docker/test_entrypoint_extract.py

```diff
@@ -0,0 +1,66 @@
+"""Test the extract entrypoint function."""
+
+import unittest
+from pathlib import Path
+import shutil
+import tempfile
+from entrypoint import extract
+
+
+class TestExtractFunction(unittest.TestCase):
```

Comment:
Please combine this file with `entrypoint_test.py` then rename it to `test_entrypoint.py`.